.circleci/config.yml

```diff
@@ -31,6 +45,18 @@ jobs:
       - install_dependencies
       - run_tests
       - upload_coverage
+  test-nightly:
```

Comment:
copy-pasta from `test`

Reply:
i think we use underscores _ everywhere else

Reply:
```suggestion
  test_nightly:
```